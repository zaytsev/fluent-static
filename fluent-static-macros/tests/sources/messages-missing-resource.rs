use fluent_static::message_bundle;

#[message_bundle(
    resources=[
        ("tests/resources/simple-en.ftl", "en"),
        ("tests/resources/simple-it.ftl", "it")], 
    default_language = "en")
]
struct Messages;

fn main() {
    let messages = Messages::default();
    assert_eq!("en foo", messages.test_param("foo"));
}
