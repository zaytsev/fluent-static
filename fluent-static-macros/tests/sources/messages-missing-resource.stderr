error: Error processing resource: Error reading resource $DIR/tests/resources/simple-it.ftl
 --> tests/sources/messages-missing-resource.rs:6:48
  |
6 |         ("tests/resources/simple-it.ftl", "it")],
  |                                                ^

error[E0433]: failed to resolve: use of undeclared type `Messages`
  --> tests/sources/messages-missing-resource.rs:12:20
   |
12 |     let messages = Messages::default();
   |                    ^^^^^^^^ use of undeclared type `Messages`
   |
help: consider importing this struct
   |
1  + use std::os::unix::net::Messages;
   |
